include/eve/algo/concepts.hpp

```diff
@@ -8,9 +8,23 @@
 #pragma once
 
 #include <concepts>
+#include <type_traits>
+#include <eve/function/load.hpp>
 
 namespace eve::algo
 {
+  namespace detail
+  {
+    template <typename>
+    struct is_eve_fixed  : std::false_type {};
```

Comment:
Not a fan of using eve_ in name of traits.

Reply:
Did `is_fixed`. It's in detail`